frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -248,49 +269,74 @@ const NewNetworkTrafficUsage: FC = () => {
-                                >
+                                <BoldText>Heads up!</BoldText> You are currently
+                                consuming more requests than your plan includes
+                                and will be billed according to our terms.
+                                Please see{' '}
+                                <RouterLink to='https://www.getunleash.io/pricing'>
                                     this page
-                                </Link>{' '}
+                                </RouterLink>{' '}
                                 for more information. In order to reduce your
                                 traffic consumption, you may configure an{' '}
-                                <Link
-                                    component={RouterLink}
-                                    to='https://docs.getunleash.io/reference/unleash-edge'
-                                >
+                                <RouterLink to='https://docs.getunleash.io/reference/unleash-edge'>
                                     Unleash Edge instance
-                                </Link>{' '}
+                                </RouterLink>{' '}
                                 in your own datacenter.
                             </Alert>
                         }
                     />
                     <StyledBox>
                         <NewHeader>
-                            {
-                                // todo: add new usage plan summary that works for monthly data as well as daily
-                            }
-
-                            <NetworkTrafficUsagePlanSummary
-                                usageTotal={usageTotal}
-                                includedTraffic={includedTraffic}
-                                overageCost={overageCost}
-                                estimatedMonthlyCost={estimatedMonthlyCost}
-                            />
+                            <TrafficInfoBoxes>
+                                <RequestSummary
+                                    period={newPeriod}
+                                    usageTotal={
+                                        newPeriod.grouping === 'daily'
+                                            ? usageTotal
+                                            : averageTrafficPreviousMonths(
+                                                  Object.keys(endpointsInfo),
+                                                  traffic.usage,
+                                              )
+                                    }
+                                    includedTraffic={includedTraffic}
+                                />
+                                {newPeriod.grouping === 'daily' &&
+                                    includedTraffic > 0 &&
+                                    usageTotal - includedTraffic > 0 &&
+                                    estimateTrafficDataCost && (
```

Comment:
🤮 Yeah, this looks horrible. I'll extract it into a separate variable in the follow-up.